CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [v0.9.1] - 2025-08-19
```

Comment:
I'd say the same to you here, [what I was told for 0.9.0](https://github.com/rust-embedded/heapless/pull/556#discussion_r2062663382). :)